acme/acme/client.py

```diff
@@ -509,15 +518,19 @@ class ClientNetwork(object):  # pylint: disable=too-many-instance-attributes
     JSON_ERROR_CONTENT_TYPE = 'application/problem+json'
     REPLAY_NONCE_HEADER = 'Replay-Nonce'
 
-    def __init__(self, key, alg=jose.RS256, verify_ssl=True,
-                 user_agent='acme-python', timeout=DEFAULT_NETWORK_TIMEOUT):
+    # pylint: disable=too-many-arguments
+    def __init__(self, key, account=None, alg=jose.RS256, verify_ssl=True,
```

Comment:
While I care less about the usage of `acme_version` in this class than I do in `Client`, if we take the separate class approach for `Client`, it might make sense to do that here as well for consistency. We could put these in a file called `client2.py` if we wanted.